modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1257,6 +1259,14 @@ Where `<expr>` is any Cypher expression and `<TYPE>` is one of the following typ
 * `ZONED DATETIME`
 * `DURATION`
 * `POINT`
+* `NODE`
+* `RELATIONSHIP`
+* `MAP`
+* `LIST<TYPE>`
```

Comment:
I got stuck with this one, adoc didn't let me do LIST<<TYPE>>, ideas on how to show this?